### ISSUE TITLE
Accessibility bug: Visible keyboard focus missing

### ISSUE THREAD

**Issue by heidiv** [lubusIN/blablablocks-tabs-block#40]
### Description

Using the default twenty twenty-five theme, tab focus is not visible. 

The culprit appears to be:
.blablablocks-tab-container .blablablock-tab-btn:focus {
    outline: none;
}

### Steps to Reproduce

Tab to tabs via keyboard

### Expected Behaviour

There should be visible focus for tabs
